index.html

```diff
@@ -0,0 +1,695 @@
+            <!-- BUTTONS -->
+            <div class="component" id="buttons-id">
+
+                <h1>Buttons</h1>
+                <p class="lead">Buttons are a middleman between the user and the product, and are charged with keeping
+                    the conversation between person and machine going.</p>
+
+                <!-- FILLED BUTTONS -->
+                <div class="category">
+                    <button class="btn btn-primary">Primary</button>
+                    <button class="btn btn-secondary">Secondary</button>
+                    <button class="btn btn-info">Info</button>
+                    <button class="btn btn-success">Success</button>
+                    <button class="btn btn-warning">Warning</button>
+                    <button class="btn btn-danger">Danger</button>
+                </div>
+
+                <div class="display-none">
+
+                    <div onclick="showCode('filled-button')" class="pre-div">
+                        <i class="fas fa-angle-double-right"></i>
+                    </div>
+
+                    <div id='filled-button' class="pre-tag" style="display: none;">
+                        <iframe src="https://carbon.now.sh/embed/AJSv4M1dMHoN1To8EB9s"
+                            style="width: 100%; height: 348px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+                </div>
+                <!-- OUTLINED BUTTONS -->
+                <div class="category">
+                    <button class="btn btn-outline-primary">Primary</button>
+                    <button class="btn btn-outline-secondary">Secondary</button>
+                    <button class="btn btn-outline-info">Info</button>
+                    <button class="btn btn-outline-success">Success</button>
+                    <button class="btn btn-outline-warning">Warning</button>
+                    <button class="btn btn-outline-danger">Danger</button>
+                </div>
+
+
+
+                <div class="display-none">
+
+                    <div onclick="showCode('outlined-buttons')" class="pre-div">
+                        <i class="fas fa-angle-double-right"></i>
+                    </div>
+
+                    <div class="pre-tag" id='outlined-buttons' style="display: none;">
+                        <iframe src="https://carbon.now.sh/embed/WgfEDndw3K0SFMAXnKYq"
+                            style="width: 100%; height: 330px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+                </div>
+
+            </div>
+            <!-- AVATAR -->
+            <div class="component" id="avatars-id">
+                <h1>Avatar</h1>
+                <p>An avatar is a representation of who we are or our fictional character. </p>
```

Comment:
class lead is missing and also not consistently used, please cross-check in all components

Reply:
done